Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -348,17 +351,40 @@ public class EOSSessionsManager : IEOSSubManager
         private SessionDetails JoiningSessionDetails = null;
         private ulong JoinedSessionIndex = 0;
 
+        /// <summary>
+        /// When joining a Session you did not create, the Session's name is not shared with you.
+        /// Instead the Session is given a local name, starting with this, when <see cref="GenerateJoinedSessionName(bool)"/> is called.
+        /// This local name doesn't have to follow any particular formatting, but does need to be locally unique and consistent.
+        /// </summary>
         private const string JOINED_SESSION_NAME = "Session#";
         private const uint JOINED_SESSION_NAME_ROTATION_NUM = 9;
         private const int JOINED_SESSION_NAME_ROTATION = 9;
+
+        /// <summary>
+        /// The Bucket Id to create Sessions under, and to search by when finding Sessions.
+        /// Bucket Ids are used to separate search results, operating as a major filter option.
+        /// Your application could use different Bucket Ids for Sessions. If you do,
+        /// then you will need to search by those Bucket Ids in order to find those Sessions online.
+        /// 
+        /// https://dev.epicgames.com/docs/en-US/api-ref/functions/eos-session-modification-set-bucket-id
+        /// </summary>
         private const string BUCKET_ID = "SessionSample:Region";
 
+        /// <summary>
+        /// When creating a Session or searching for one, this is the attribute used to filter for buckets.
+        /// In these sample functions, <see cref="CreateSession(Session, bool, Action)"/> sets the BucketId
+        /// in <see cref="CreateSessionModificationOptions"/> as well as an <see cref="SessionAttribute"/>.
+        /// </summary>
         private const string EOS_SESSIONS_SEARCH_BUCKET_ID = "bucket";
+
+        // TODO: All three of these are unused, remove?
```

Comment:
nit: I would categorize this as a `NOTE:`, the `TODO:` version would be "Confirm that the three proceeding values are unused, and if so remove them."

Reply:
This is definitely meant to be followed up on, so I'll rephrase this to be more like a TODO comment.